crates/walrus-service/bin/node.rs

```diff
@@ -191,6 +191,27 @@ enum AdminCommands {
         #[command(subcommand)]
         command: CheckpointCommands,
     },
+    /// Restore the database from a checkpoint.
+    Restore {
+        /// The path where the checkpoint will be created. If not specified, the checkpoint will be
+        /// created in the `checkpoint_dir` specified in [`StorageNodeConfig::checkpoint_config`].
+        #[arg(long)]
+        #[serde(default)]
+        db_checkpoint_path: PathBuf,
```

Comment:
Question: can this field not be specified? Should it be `Option<PathBuf>`?

Reply:
Sorry I was not clear. The doc says "If not specified, ...". Maybe just remove that part from the doc.